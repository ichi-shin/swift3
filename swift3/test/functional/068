#!/bin/bash
# Copyright(c)2014 NTT corp. All Rights Reserved.
# DB1-1-3-2_2

. ./common

S3USER=tester

_s3_get /no_bucket?lifecycle -D - | _filter_curl xml
