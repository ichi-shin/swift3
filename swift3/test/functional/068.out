QA output created by 068
> s3curl --id tester -- -X GET -D - http://SWIFT_HOST/no_bucket?lifecycle... 404
HTTP/1.1 404 Not Found
Content-Type: text/xml
Date: DATE
Transfer-Encoding: chunked
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

<?xml version='1.0' encoding='UTF-8'?>
<Error>
  <Code>NoSuchBucket</Code>
  <Message>The specified bucket does not exist.</Message>
  <RequestId>TXID</RequestId>
  <BucketName>no_bucket</BucketName>
</Error>
